npm/webpack-preprocessor/package.json

```diff
@@ -22,6 +22,8 @@
   "dependencies": {
     "bluebird": "3.7.1",
     "debug": "^4.3.2",
+    "del": "^6.1.1",
```

Comment:
Well has quite a few dependencies to only use it to delete the temp dir. Why did you choose `del` vs `fs-extra` (fewer dependencies) or just using fs provided by node?

Reply:
Node's `fs` doesn't have support for recursively removing directories until 14.4.0 and we need to support Node 12+. Good call on using `fs-extra` instead of `del`. Looks like it [it is smaller](https://bundlephobia.com/package/fs-extra@10.1.0) than [del](https://github.com/sindresorhus/del). Changed it in https://github.com/cypress-io/cypress/pull/23283/commits/51492f3d3ef2fe7799b5c5ec5be4ef938602bea9